Furball.Vixie.Backends.OpenGL41/QuadRendererGL41.cs

```diff
@@ -71,8 +71,8 @@ private struct InstanceData {
 
         public unsafe QuadRendererGL41(OpenGL41Backend backend) {
             this._backend = backend;
+            this._backend.CheckError("aaa");
```

Comment:
why?

Reply:
accidentally left in from when i was tracking down a gl error